src/main/java/seedu/condonery/logic/parser/CondoneryParser.java

```diff
@@ -6,15 +6,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.condonery.logic.commands.AddCommand;
-import seedu.condonery.logic.commands.ClearCommand;
-import seedu.condonery.logic.commands.Command;
-import seedu.condonery.logic.commands.DeleteCommand;
-import seedu.condonery.logic.commands.EditCommand;
-import seedu.condonery.logic.commands.ExitCommand;
-import seedu.condonery.logic.commands.FindCommand;
-import seedu.condonery.logic.commands.HelpCommand;
-import seedu.condonery.logic.commands.ListCommand;
+import seedu.condonery.logic.commands.*;
```

Comment:
For this part, I recall that the coding standards taught by the module tell us to always import classes explicitly. What do you think?

Reply:
Oh yes, the auto-import made the change. I have fixed it. Thanks!